plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/UIUtils.java

```diff
@@ -1671,6 +1671,34 @@ public static IWorkbenchWindow getParentWorkbenchWindow(Control control) {
         return null;
     }
 
+    /**
+     * Starting from root shell iterates through visible child shells to find
+     * shell without any visible children
+     * see #13691
+     * @return child shell without visible children
+     */
+    @Nullable
+    public static Shell getTopActiveWorkbenchShell() {
```

Comment:
I'm not sure if it's a good decision to put this method here. It currently has only one usage, and UIUtils is already quite bloated.